packages/worker/src/sdk/users/users.ts

```diff
@@ -257,6 +261,17 @@ export const save = async (
     }
   }
 
+  let appsToRemove: string[] = []
+  if (dbUser && instanceOfUser(user)) {
+    const newRoles = Object.keys(user.roles)
+    const existingRoles = Object.keys(dbUser.roles)
+
+    appsToRemove = existingRoles.filter(r => !newRoles.includes(r))
+    if (appsToRemove.length) {
+      console.log("Deleting access to apps", { appsToRemove })
```

Comment:
Generally we try not to leave console logs in API requests, if there is useful information here that could be logged in the form of a warning or error then we would use our logging functionality (which we can detect easily in Elastic) under `import { logging } from "@budibase/backend-core"` e.g. `logWarn` or `logAlert`.